CONTRIBUTING.md

```diff
@@ -1,89 +1,90 @@
-# Coding guidelines
+# Contirbution guidelines
```

Comment:
This is what I was expecting. Can you take a look at the changes and resolve this comments once you've done so.